rest/src/main/resources/META-INF/faces-config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='UTF-8'?>
+
+<faces-config version="2.3"
+              xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-facesconfig_2_3.xsd">
+</faces-config>
```

Comment:
Please use JSF 3.0 one.

Reply:
Done

Reply:
Looks like you only adjusted the one in `/cdi`. Please also adjust abovementioned one :)

Reply:
Done